--- telepathy-mission-control-5.9.1/configure.ac~	2011-07-21 14:08:55.000000000 +0200
+++ telepathy-mission-control-5.9.1/configure.ac	2011-09-02 08:45:50.759360074 +0200
@@ -340,7 +340,7 @@
 AC_SUBST(TELEPATHY_CFLAGS)
 
 PKG_CHECK_MODULES([GLIB],
-	[glib-2.0 >= 2.28, gobject-2.0, gmodule-no-export-2.0])
+	[glib-2.0 >= 2.28, gio-2.0, gobject-2.0, gmodule-no-export-2.0])
 AC_SUBST(GLIB_LIBS)
 AC_SUBST(GLIB_CFLAGS)
 
